new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -136,7 +155,20 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
     // create standard, reusable retry/backoff policy
     withStandardRetryPolicy = with().pollDelay(2, SECONDS)
         .and().with().pollInterval(10, SECONDS)
-        .atMost(5, MINUTES).await();
+        .atMost(6, MINUTES).await();
+
+    // create a reusable quick retry policy
+    withQuickRetryPolicy = with().pollDelay(0, SECONDS)
+        .and().with().pollInterval(4, SECONDS)
+        .atMost(10, SECONDS).await();
+
+    // clean up the download directory so that we always get the latest
+    // versions of the tools in every run of the test class.
+    try {
+      cleanupDirectory(DOWNLOAD_DIR);
```

Comment:
I think we want to clean up the directory once before all tests run. This can be moved to ImageBuilders.beforeAll()?

Reply:
Moved. Thanks.